server/modules/activityLog/repository/activityLogRepository.js

```diff
@@ -2,11 +2,12 @@ import camelize from 'camelize'
 
 import { getSurveyDBSchema } from '@server/modules/survey/repository/surveySchemaRepositoryUtils'
 import * as User from '@core/user/user'
-import * as db from '@server/db/db'
-import * as DbUtils from '@server/db/dbUtils'
 
 import * as ActivityLog from '@common/activityLog/activityLog'
 
+import * as db from '@server/db/db'
+import * as DbUtils from '@server/db/dbUtils'
+
 //===== CREATE
 export const insert = async (user, surveyId, type, content, system, client) =>
   await client.none(`
```

Comment:
can you add assert checks here?

Reply:
ActivityLogRepository is used all over the application, I wouldn't do any check there.